test/dispatcher.js

```diff
@@ -62,7 +75,7 @@ suite('Event Generation and Dispatching', function() {
 
   test('PointerEvents from mouse fire anywhere by default', function() {
     // move always fires
-    var cb = chai.spy();
+    var cb = chai.spy;
```

Comment:
As a factory, this line throws "TypeError: chai.spy is not a function", perhaps that is symptomatic of another problem?